.github/workflows/linux.yaml

```diff
@@ -46,3 +46,11 @@ jobs:
     with:
       platform: linux-gcc-6-3
       nightly: ${{ github.event.inputs.nightly }}
+  linux-coverage:
```

Comment:
This won't run on nightly basis, but on postsubmit only.  You want something like:

inputs.nightly == 'true' || github.event_name == 'schedule'

Reply:
I changed my mind. I think it makes more sense to run this on each postsubmit on main. That way we get continuous updating of coverage data instead of a daily (nightly) update.